### DISCUSSION TITLE
Mate terminal port

### DISCUSSION THREAD

**Discussion by albicasty** [catppuccin/catppuccin#2598] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

Hello, I ported catppuccin for mate-terminal./dconf Let me know if you are interested into publishing it, so that I will make the repository ready to be published.

Here is the link: https://github.com/albicasty/catppuccin-mate

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hey 👋,

We recently updated our ANSI colours in v1.7.1 so we'd want the terminal colours to match that.

I can see that you've taken inspiration from our gnome-terminal port. I recently updated that script to pull in the new ANSI colours (https://github.com/catppuccin/gnome-terminal/blob/main/install.py) so I think I'd be happy to review a mate terminal port if it followed our new ANSI colours 👍
